### ISSUE TITLE
No redirect when clicking "submit" in User Preferences

### ISSUE THREAD

**Issue by ciur** [ciur/papermerge#490]
Browsers: all
Backend version: 2.1.0b10
Frontend version: 2.1.0b8

### Description

When user clicks "submit" button in preferences menu - no redirect or notification with "Preferences changes were saved" i.e. there is no any feedback that changes took effect. On the other hand when user clicks cancel - he/she is redirected to Preferences index:

![Peek 2022-10-16 06-45](https://user-images.githubusercontent.com/24827601/196018580-d0ab216f-3521-40eb-a1d9-382cbf4f9c33.gif)

### Expected Behavior

When user clicks "submit" in preferences he/she must be redirected to preferences index and there should be a notification saying that "changes were saved".

**Comment by Aysha-py** [comment]
@ciur 
is this issue still available to be contributed to?

**Comment by ciur** [comment]
@Aysha-py yes, issue is still open. The problem is on the [frontend side|https://github.com/papermerge/papermerge.js]
